api/src/services/cereNetworkService.js

```diff
@@ -1,15 +1,26 @@
 const { ApiPromise, WsProvider, Keyring } = require("@polkadot/api");
-const { config, CERE_MAINNET, CERE_MAINNET_WS_PROVIDER_URL_DEFAULT, MNEMONIC_EMPTY } = require("../constants/config");
-const {
-  NETWORKS,
-  CERE_WS_PROVIDER_URL
-} = process.env;
+const { cereTypes, blockchains } = require("../config");
+const { NETWORKS } = process.env;
 
 const networkParams = new Map();
 
 async function init() {
-  networkParams.set(CERE_MAINNET, await initNetwork(CERE_WS_PROVIDER_URL || CERE_MAINNET_WS_PROVIDER_URL_DEFAULT, MNEMONIC_EMPTY));
+  const promises = [];
+  const cere = blockchains[0];
+  cere.networks.forEach(network => {
+    const promise = async() => ({
+      name: network.name,
+      rpc: await initNetwork(network.rpcUrl, network.mnemonic)
+    });
+    promises.push(promise());
+  });
+  const res = await Promise.all(promises);
+  res.forEach(network => {
+    networkParams.set(network.name, network.rpc);
+  });
 
+  // Deprecated: init networks from NETWORKS config
+  // ToDo: remove it in the future from code and devops configs
```

Comment:
Let's rename `devops configs` to `deployment configs`

Reply:
resolved